views/default/input/access.php

```diff
@@ -66,7 +66,7 @@
 }
 
 // don't call get_default_access() unless we need it
-if (!isset($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
+if (!isset($vars['value']) || is_null($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
```

Comment:
I think it'd be clearer to add a line of code above: `$vars['value'] = (int) elgg_extract('value', $vars);`
That leaves two simple checks: `=== null` or `== ACCESS_DEFAULT`.